core/templates/pages/topic-viewer-page/topic-viewer-page.component.css

```
@@ -7,9 +7,151 @@
+
+topic-viewer-page .redesigned-secondary-section subtopics-list .oppia-page-card.oppia-long-text,
+topic-viewer-page .redesigned-secondary-section practice-tab .oppia-page-card {
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+}
+
+topic-viewer-page .topic-viewer-page-content .tab-title {
+  color: #017a70;
+  font-size: 2em;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)